packages/vm/src/errors.rs

```diff
@@ -269,6 +269,7 @@ impl CommunicationError {
         InvalidOrder { value }.build()
     }
 
+    #[allow(dead_code)]
     pub(crate) fn invalid_utf8<S: ToString>(msg: S) -> Self {
```

Comment:
I think that if you add the utf-8 checking in the VM side like i described, you can remove this annotation

Reply:
Urg, this is squeezed int an `FfiError::Other`:

```rust
        let human =
            String::from_utf8(result).map_err(|_| FfiError::other("Result was not valid utf-8"))?;
        Ok(HumanAddr(human))
```

Reply:
oh yeah, that's right :/
I guess that's why you suggested the UTF variant in #445.